webapp/components/survey/SurveyCreate/store/useCreateSurvey.js

```diff
@@ -40,11 +41,20 @@ export const useCreateSurvey = ({ template = false } = {}) => {
     setNewSurvey(newSurveyUpdated)
   }
 
+  const onUploadProgress = useCallback(
```

Comment:
maybe this can be passed to the Actions instead of propagate to the component,

Reply:
right, I'll change it